src/components/TaskRunnerPane/TaskRunnerPane.js

```diff
@@ -28,19 +28,17 @@ class TaskRunnerPane extends Component<Props, State> {
     selectedTaskId: null,
   };
 
-  static getDerivedStateFromProps(props, state) {
+  componentWillReceiveProps(nextProps) {
```

Comment:
Why the switch away from the the static method?

Reply:
I realized since posting this that it's unrelated to this change, it's happening to me in master as well :/

But yeah, without this change I get this warning now:
<img width="749" alt="screen shot 2018-08-25 at 3 07 29 pm" src="https://user-images.githubusercontent.com/6692932/44621691-aa35e200-a878-11e8-80f5-5c3c80d1eec6.png">

In addition to this warning, I get a crash:
<img width="575" alt="screen shot 2018-08-25 at 3 08 25 pm" src="https://user-images.githubusercontent.com/6692932/44621694-bae65800-a878-11e8-8664-2d3311e8bec3.png">

I realized the issue appears to be that Redux' HOC (provided by `connect`) implements `componentWillReceiveProps`, and that method is no longer being called (which is funny to me since I thought a HOC would be a completely separate component). Because of that, the `tasks` that redux provides isn't being passed down to the component, and it's blowing up.

Note that `mapStateToProps` will _never_ return `undefined` for tasks. Indeed, when I put a console.log in it, it isn't even being called.

So yeah, while I don't really understand why this is an issue, it seems like a wider issue affecting these particular versions of react and react-redux, so I'm not too worried.

Actually, it'd be good to try updating react-redux to see if that fixes it... I'll give that a go.

Reply:
Ah, we're already on the latest react-redux.

Oh well. I'm not worried, I'm sure they'll get it fixed by the time `componentWillReceiveProps` is deprecated